vizro-ai/changelog.d/20241016_133757_antony.milne_bump_python_py39.md

```diff
@@ -0,0 +1,47 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Highlights ✨
+
+- A bullet item for the Highlights ✨ category with a link to the relevant PR at the end of your entry, e.g. Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))
+
+-->
+
+### Removed
+
+- Drop support for Python 3.8. ([#813](https://github.com/mckinsey/vizro/pull/813))
+
```

Comment:
Do we have this change, since the `requires-python = ">=3.9"` is already set in the pyproject.toml for vizro-ai?

Reply:
Oops, no we don't! You're quite right - I will remove this.

Reply:
[63d4a28](https://github.com/mckinsey/vizro/pull/813/commits/63d4a28b8455d14d2b96408db4149ebd34791cde)